test/commands/scanner/rule/remove.test.ts

```diff
@@ -1,16 +1,17 @@
-import {expect, test} from '@salesforce/command/lib/test';
+import {expect} from '@salesforce/command/lib/test';
+import {setupCommandTest} from '../../../TestUtils';
 import {Messages} from '@salesforce/core';
 import {Controller} from '../../../../src/Controller';
-import * as TestOverrides from '../../../test-related-lib/TestOverrides';
 import { CUSTOM_PATHS_FILE } from '../../../../src/Constants';
 import fs = require('fs');
 import path = require('path');
 
 Messages.importMessagesDirectory(__dirname);
 const messages = Messages.loadMessages('@salesforce/sfdx-scanner', 'remove');
 
-TestOverrides.initializeTestSetup();
-const SFDX_SCANNER_PATH = Controller.getSfdxScannerPath();
```

Comment:
Calling this before the IOC container is initialized is problematic. Make it lazy.